src/test/java/in/projecteka/consentmanager/link/link/LinkUserJourneyTest.java

```diff
@@ -62,6 +63,7 @@
 import static java.util.List.of;
 import static org.mockito.Mockito.when;
 
+@ActiveProfiles("default")
```

Comment:
Instead of using this
 @ConditionalOnProperty(value = "cm.cacheMethod", havingValue = "gauva", matchIfMissing = true) for gauva cache
@ConditionalOnProperty(value = "cm.cacheMethod", havingValue = "redis")
this way we don't need to rely on the profiles or anything. By default everything should work as it's.

Reply:
Actually this is not needed here. Reason I've added is to use it as a placeholder. For after making the switch for default profile to use redis, this has to be updated to some other profile like dev. The profile way would mean no need for configuring redis details(or configuring the new cachemethod) for local setup.

Reply:
> The profile way would mean no need for configuring redis details(or configuring the cachemethod property) for local setup.
That should work with conditionalOnProperty also no?

Reply:
Yes, With additional property having to be updated like the `cm.CacheMethod`. This introduces multiplicity.